docs/pages/contribute/development.md

```diff
@@ -0,0 +1,70 @@
+{% include navigation.html %}
+
+# How to setup a development machine
+
+## Install development software
+
+First, install the necessary development software on your machine:
+* IntelliJ > [link](https://www.jetbrains.com/idea)
+* IntelliJ plugins: Lomkok > [link](https://plugins.jetbrains.com/plugin/6317-lombok)
+* Hub - command line tool for git > [link](https://github.com/github/hub)
```

Comment:
Hub is only needed for actual contribution not for creating an IESI instance. Maybe move this software to a dedicated section in pages/contribute/development/contribute.md

Reply:
Agreed. Remove the sentence for now. To be added in resolution for #121

Reply:
Resolved using commit https://github.com/metadew/iesi/pull/119/commits/cc259ee8e39120af8d92c1763b0b1ff064621e23